ONSdigital/sdc-int-rh-ui
# 671 Add invalid length and malformed UAC error message translation

# Motivation and Context

We need to add translations for invalid length and malformed UACs
# What has changed



* I've moved the logic around to get this to work but have made very, very (very) light touches of changes for MVP - we want to keep the logic and validation as it was, and any major refactoring is out of scope for now.
* Move the check for UAC length to a standalone check so that we can display new error message text to the user
* Add UAC length constant to avoid a magic number

# How to test?



Run locally, test bad UACs and see if the errors are as expected. Run the ATs and make sure they pass
# Links



https://trello.com/c/BWcmiMWY
# Screenshots (if appropriate):
